sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/DigitalTwinsRestClient.cs

```diff
@@ -585,6 +587,142 @@ internal Response<Stream> Add(string id, Stream twin, CreateOrReplaceDigitalTwin
             }
         }
 
+        internal async Task<Response<RelationshipCollection<T>>> ListRelationshipsAsync<T>(string id,
+            string relationshipName = null,
+            GetRelationshipsOptions digitalTwinsListRelationshipsOptions = null,
+            ObjectSerializer objectSerializer = null,
+            CancellationToken cancellationToken = default)
+        {
+            if (id == null)
+            {
+                throw new ArgumentNullException(nameof(id));
+            }
+
+            if (objectSerializer == null)
+            {
+                throw new ArgumentNullException(nameof(objectSerializer));
+            }
+
+            using HttpMessage message = CreateListRelationshipsRequest(id, relationshipName, digitalTwinsListRelationshipsOptions);
+            await _pipeline.SendAsync(message, cancellationToken).ConfigureAwait(false);
+            switch (message.Response.Status)
+            {
+                case 200:
+                    {
+                        using JsonDocument document = await JsonDocument.ParseAsync(message.Response.ContentStream, default, cancellationToken).ConfigureAwait(false);
+                        RelationshipCollection<T> value = RelationshipCollection<T>.DeserializeRelationshipCollection(document.RootElement, objectSerializer);
+                        return Response.FromValue(value, message.Response);
+                    }
+                default:
+                    throw await _clientDiagnostics.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);
+            }
+        }
+
+        internal async Task<Response<RelationshipCollection<T>>> ListRelationshipsNextPageAsync<T>(string nextLink,
```

Comment:
first parameter on newline (unless this is how autorest formats this code and this is just a copy and minor edit)